tools/eksDistroBuildToolingOpsTools/cmd/eksGoTool/cmd/backportIssue.go

```diff
@@ -0,0 +1,162 @@
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			label := []string{"golang", "security"}
+			assignee := "rcrozean"
+			issueState := "open"
+
+			retrier := retrier.New(time.Second*380, retrier.WithBackoffFactor(1.5), retrier.WithMaxRetries(15, time.Second*30))
+
+			token, err := github.GetGithubToken()
+			if err != nil {
+				return fmt.Errorf("getting Github PAT from environment at variable %s: %v", github.PersonalAccessTokenEnvVar, err)
+			}
+			githubClient, err := github.NewClient(cmd.Context(), token)
+			if err != nil {
+				return fmt.Errorf("setting up Github client: %v", err)
+			}
+
+			if backportVersions[0] == "" {
+				// set up Repo Content Creator handler
+				repoContentManagerOpts := &repoContentManager.Opts{
+					SourceOwner: "rcrozean",
+					SourceRepo:  "eks-distro-build-tooling",
+				}
+				rm := repoContentManager.New(retrier, githubClient, repoContentManagerOpts)
+
+				gfOpts := &repoContentManager.GetFileOpts{
+					Owner: "aws",
+					Repo:  "eks-distro-build-tooling",
+					Path:  eksGoSupportedVersionsPath,
+					Ref:   nil,
+				}
+
+				test, err := rm.GetFile(cmd.Context(), gfOpts)
+				if err != nil {
+					return fmt.Errorf("getting file at %s: %v", gfOpts.Path, err)
+				}
+				testContent, err := test.GetContent()
+				if err != nil {
+					return fmt.Errorf("getting file content from %v: %v", test.Name, err)
+				}
+
+				backportVersions = strings.Split(testContent, "\n")
+			}
+
+			// set up Issue Creator handler
+			issueManagerOpts := &issueManager.Opts{
+				SourceOwner: "rcrozean",
+				SourceRepo:  "eks-distro-build-tooling",
+			}
+			im := issueManager.New(retrier, githubClient, issueManagerOpts)
+
+			giOpts := &issueManager.GetIssueOpts{
+				Owner: "aws",
+				Repo:  "eks-distro-build-tooling",
+				Issue: toplevelIssueId,
+			}
+
+			toplevelIssue, err := im.GetIssue(cmd.Context(), giOpts)
+			if err != nil {
+				return fmt.Errorf("getting toplevel issue %v from %v: %v", giOpts.Issue, giOpts.Repo, err)
+			}
+
+			for _, ver := range backportVersions {
+				if ver != "" {
+					issueOpts := &issueManager.CreateIssueOpts{
+						Title:    GenerateBackportIssueTitle(toplevelIssue, ver),
+						Body:     GenerateBackportIssueBody(toplevelIssue, ver),
+						Labels:   &label,
+						Assignee: &assignee,
+						State:    &issueState,
+					}
+
+					if !dryRun {
+						_, err := im.CreateIssue(cmd.Context(), issueOpts)
+						if err != nil {
+							return fmt.Errorf("creating issue: %v", err)
+						}
+					}
+				}
+			}
+
+			return nil
+		},
+	}
+)
+
+func init() {
+	backportCmd.AddCommand(backportIssueCmd)
+	backportIssueCmd.Flags().IntVarP(&toplevelIssueId, toplevelIssueIdFlag, "i", 0, "Upstream Issue ID e.g. ")
+	backportIssueCmd.Flags().StringSliceVarP(&backportVersions, backportVersionFlag, "b", []string{""}, "to specify versions to backport use this flag. Multiple versions can be specified separated by commas. e.g. <ver>,<ver>,<ver>")
```

Comment:
my preference is to just not default this and then do a nil check for it later rather than look for the empty string, I don't feel super strongly about it tho